src/base/bittorrent/torrentinfo.cpp

```diff
@@ -338,16 +355,16 @@ TorrentInfo::PieceRange TorrentInfo::filePieces(int fileIndex) const
     }
 
     const libt::file_storage &files = nativeInfo()->files();
-    const auto fileSize = files.file_size(fileIndex);
-    const auto fileOffset = files.file_offset(fileIndex);
-    return makeInterval(static_cast<int>(fileOffset / pieceLength()),
-                        static_cast<int>((fileOffset + fileSize - 1) / pieceLength()));
+    const auto fileSize = files.file_size(makeFileIndex(fileIndex));
+    const auto firstOffset = files.file_offset(makeFileIndex(fileIndex));
```

Comment:
Seems like rebasing issue. firstOffset was changed to fileOffset in master some time ago.